examples/acados_matlab_octave/getting_started/minimal_example_closed_loop.m

```diff
@@ -70,44 +70,51 @@
 nx = model.nx;
 nu = model.nu;
 
-%% model to create the solver
+%% acados ocp model
 ocp_model = acados_ocp_model();
 model_name = 'pendulum';
-
-%% acados ocp model
 ocp_model.set('name', model_name);
 ocp_model.set('T', T);
+
 % symbolics
 ocp_model.set('sym_x', model.sym_x);
 ocp_model.set('sym_u', model.sym_u);
 ocp_model.set('sym_xdot', model.sym_xdot);
 
 % nonlinear-least squares cost
+ocp_model.set('cost_type_0', 'nonlinear_ls');
 ocp_model.set('cost_type', 'nonlinear_ls');
 ocp_model.set('cost_type_e', 'nonlinear_ls');
 
+ocp_model.set('cost_expr_y_0', model.cost_expr_y_0);
+ocp_model.set('cost_W_0', model.cost_W_0);
 ocp_model.set('cost_expr_y', model.cost_expr_y);
+ocp_model.set('cost_W', model.cost_W);
 ocp_model.set('cost_expr_y_e', model.cost_expr_y_e);
+ocp_model.set('cost_W_e', model.cost_W_e);
 
-W_x = diag([1e2, 1e2, 1e-2, 1e-2]);
-W_u = 1e-3;
-W = blkdiag(W_x, W_u);
-ocp_model.set('cost_W', W);
-ocp_model.set('cost_W_e', model.W_e);
+% intiialize reference to zero, change later
+ocp_model.set('cost_y_ref_0', zeros(size(model.cost_expr_y_0)));
+ocp_model.set('cost_y_ref', zeros(size(model.cost_expr_y)));
+ocp_model.set('cost_y_ref_e', zeros(size(model.cost_expr_y_e)));
 
 % dynamics
 ocp_model.set('dyn_type', 'explicit');
-ocp_model.set('dyn_expr_f', model.expr_f_expl);
+ocp_model.set('dyn_expr_f', model.dyn_expr_f_expl);
 
 % constraints
 ocp_model.set('constr_type', 'auto');
-ocp_model.set('constr_expr_h', model.expr_h);
+ocp_model.set('constr_expr_h_0', model.constr_expr_h_0);
```

Comment:
I do see that this is an important detail.
But, the goal of the examples in `getting_started` is really to make them as easy as possible to not overwhelm new users.
Thus, I would suggest to add the "simpler version" of the pendulum model file to the `getting_started` folder, i.e. ones where the initial node has the same cost and constraints as the terminal one and just have the one with the separate initial shooting node formulation in `pendulum_on_cart_model`.

What do you think?
Could you take care of this?

Reply:
On a second thought, the differentiation between `h` and `h_0` is pretty important, and I would keep them also in the getting started examples.
Maybe, we can just get rid of the additional `y_0` definition, an additional cost on `x_0` doesnt really hurt and it simplifies the code quite when we omit it.

Reply:
What do you think about leaving the `y_0` etc. in the example as it is for now, and possibly removing them when the documentation is updated? I think that it would help clarify the general separation of the initial stage a bit more.

Reply:
alright, let's do that. I will merge as is then :)